README.md

```diff
@@ -51,7 +65,8 @@ to be able to find calibration reference files. For more information, see the
 export jref="/grp/crds/cache/references/hst/"
 ```
 
-For more information, please see the [ACS Data Handbook](https://hst-docs.stsci.edu/acsdhb). 
+For more information, please see the [ACS Data Handbook](https://hst-docs.stsci.edu/acsdhb)
```

Comment:
Suggestion: Just after the export jref line, you could add in this single location a reminder for the user that if they are running the software on a machine which is not connected to STScI the jref needs to be set to the location where the calibration reference files exist on their own machine.

Reply:
Yes, strictly speaking, but I was hoping folks would get the general idea.  You can repeat or put the statement in a place where it is clear it applies to all the calibration reference files.